applicationset/generators/git.go

```diff
@@ -205,21 +222,32 @@ func (g *GitGenerator) filterApps(Directories []argoprojiov1alpha1.GitDirectoryG
 	return res
 }
 
-func (g *GitGenerator) generateParamsFromApps(requestedApps []string, _ *argoprojiov1alpha1.ApplicationSetGenerator) []map[string]string {
+func (g *GitGenerator) generateParamsFromApps(requestedApps []string, _ *argoprojiov1alpha1.ApplicationSetGenerator, useGoTemplate bool) []map[string]interface{} {
 	// TODO: At some point, the appicationSetGenerator param should be used
 
-	res := make([]map[string]string, len(requestedApps))
+	res := make([]map[string]interface{}, len(requestedApps))
 	for i, a := range requestedApps {
 
-		params := make(map[string]string, 2)
-		params["path"] = a
-		params["path.basename"] = path.Base(a)
-		params["path.basenameNormalized"] = sanitizeName(path.Base(a))
-		for k, v := range strings.Split(params["path"], "/") {
-			if len(v) > 0 {
-				params["path["+strconv.Itoa(k)+"]"] = v
+		params := make(map[string]interface{}, 2)
```

Comment:
`2` is kind of a weird starting size now. Want to make it `5` so it'll always fit the go template map without needing to expand?